masonry/src/widgets/grid.rs

```diff
@@ -150,27 +150,29 @@ impl GridParams {
 impl Grid {
     /// Add a child widget.
     ///
-    /// See also [`with_child`].
-    ///
-    /// [`with_child`]: Grid::with_child
+    /// See also [`with_child`](Grid::with_child).
```

Comment:
Similar thing here, that "See also" is a bit too oblique here. However, that would be part of the same aforementioned consistency pass.